code/game/turfs/simulated/outdoors/outdoors.dm

```diff
@@ -27,6 +27,11 @@ var/global/list/turf_edge_cache = list()
 	var/can_dig = FALSE
 	var/loot_count
 
+/turf/simulated/floor/outdoors/examine(var/mob/user)
+	. = ..()
+	if(!has_snow())
+		. += "A shovel could be used on help intent to dig up worms or rocks, harm intent to dig a grave, or any other intent to dig a plot for planting."
```

Comment:
For what good it normally does, do you think this should also go in the shovel's `extended_desc`?